pwa/src/components/logs/logTable.tsx

```diff
@@ -220,11 +245,19 @@ export default function LogTable({ id = null }) {
                           }
                         },
                         {
-                          title: "Context",
-                          id: "logRequestContext",
+                          title: "Content",
+                          id: "logRequestContent",
                           render: function () {
                             return (<>
-                              {log.requestContext ? JSON.stringify(log.requestContext) : <p className="utrecht-paragraph">No context found</p>}
+                              {log.requestContent
+                                ?
+                                <pre style={{ 'backgroundColor': 'black', 'overflowWrap': 'break-word' }}>
+                                  <code id="requestContent" style={{ color: 'orange' }} className="language-json">
```

Comment:
Ook deze styling wordt nu dubbel geschreven.

Reply:
Heel het content onderdeel wordt eigenlijk dubbel geschreven, misschien dat het een idee is om (in hetzelfde bestand) een extra component aan te maken e.g.:

```
const CodeContent: React.FC<..> = () => {
  // logica van dit component
}
```

En deze kan je dan tweemaal includen i.p.v. dat je de code tweemaal schrijft.